tests/Behat/Gherkin/Cucumber/CompatibilityTest.php

```diff
@@ -86,13 +121,45 @@ public function testFeaturesParseTheSameAsCucumber(\SplFileInfo $file)
         );
     }
 
+    /**
+     * @dataProvider behatFeatures
+     */
+    public function testBehatFeaturesParseTheSame(\SplFileInfo $ymlFile)
+    {
+        if (isset($this->behatFeaturesNotParsingCorrectly[$ymlFile->getFilename()])){
+            $this->markTestIncomplete($this->behatFeaturesNotParsingCorrectly[$ymlFile->getFilename()]);
+        }
+
+        if (isset($this->behatFeaturesCucumberCannotParseCorrectly[$ymlFile->getFilename()])){
+            $this->markTestIncomplete($this->behatFeaturesCucumberCannotParseCorrectly[$ymlFile->getFilename()]);
+            return;
```

Comment:
this `return` is dead code as `markTestIncomplete` never returns but always throws